### DISCUSSION TITLE
2022/06/17/hexo-latex/

### DISCUSSION THREAD

**Discussion by giscus** [argvchs/giscus-comments#24] [Announcements]
# 2022/06/17/hexo-latex/

https://argvchs.github.io/2022/06/17/hexo-latex/

**Comment by Revethere** [comment]
特殊符号无法渲染

比如 % , & 等

`$%$` 用的时候是空的

顺便问一下怎样支持颜色渲染

**Reply by argvchs** [reply]
用 KaTeX 就可以了

**Comment by tyellow07** [comment]
如果你正在使用 ParticleX 主题，可以忽略 MathJax 的……

你的那個藍色框框是怎麼用的
我如果想使用note tag 該怎麼做
